src/Wallabag/CoreBundle/Resources/translations/messages.es.yml

```diff
@@ -19,9 +19,9 @@ menu:
         unread: 'Sin leer'
         starred: 'Favoritos'
         archive: 'Archivo'
-        all_articles: 'Todos los artículos'
+        all_articles: 'Todos artículos'
         config: 'Configuración'
-        tags: 'Tags'
+        tags: 'Etiquetas'
```

Comment:
It _tags_ become _etiquetas_ could you check for other _tag_ translation in that file too?